Makefile

```diff
@@ -656,15 +655,7 @@ run-loc-tests: all-loc-tests
 	@echo '---- Run LOC unit-test: ----'
 	./$(LOC_MACRO_TEST_BIN)
 
-# ###################################################################
-#
-test-old: use-cases/single-file-C-program/test-main.c src/l3.c l3.S include/l3.h
```

Comment:
Deleted this old target that was added in very initial version of this Makefile.

All this work has been subsumed in other targets, so this is no longer needed.